### ISSUE TITLE
SALV doesn't write links correctly

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#601]
When using TRAN to restore a DUMP tape with links, the file system gets corrupted.  CHCKR output:

```LARS  ;FOO    BAR     PACK 0, UNIT #0
STARTING ADDRESS NOT SET
FILE UNPROTECTED IN OLD TUT - LARS  ;FOO    BAR     PACK 0, UNIT #0
Block 3351
LARS  ;FOO    BAR     PACK 0, UNIT #0
BLOCK POINTS OFF DSK 46333
USER DIRECTORY: LARS  ;
FOO    BAR     PACK 0, UNIT #0
        00 (INITIAL ZERO)
0       16 (SKIP 2) 213776 ?? BAD BLOCK # ??
        33 (SKIP 17) 214016 ?? BAD BLOCK # ??
        40 33 51 (JUMP 3351) 3351 ?? TUT=0??
        64 63 33 (JUMP 46333) 46333 ?? BAD BLOCK # ??
        00 (STOP)

**Comment by larsbrinkhoff** [comment]
This is about the old KA/KL SYSTEM; SALV >.

**Comment by larsbrinkhoff** [comment]
The branch [lars/salv](https://github.com/PDP-10/its/tree/lars/salv) has a SALV bug fix for reading tapes with dumped links.

**Comment by larsbrinkhoff** [comment]
Fixed by 9adfbf07888c8c5a56d372a9c93f9c4c6c5d4e2e.
